Jonathan-jonathankern/Homework-3/json-homework/json-hw.js

```diff
@@ -0,0 +1,64 @@
+
+/*
+1. Convert the string `jsonString` to a Javascript Object and store it in a variable
+called `flickrObj`
+*/
+
+let flickrObj = JSON.parse(jsonString);
+// console.log(flickrObj);
+
+/*
+2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+with the date the photo was taken. The first console.log should return the following:
+
+"Edelweiss Air, Airbus A320-214, HB-IJU 2016-02-23T18:48:14Z"
+
+*/
+
+let flickrObjItems = flickrObj["items"];
+// console.log(flickrObjItems);
+let flickrObjItemsLength = flickrObjItems.length;
+// console.log(flickrObjItemsLength);
+
+
+for(let i = 0; i < flickrObjItemsLength; i++) {
+	// console.log(i);
+	let currentItem = flickrObjItems[i];
+	// console.log(currentFlickrObjItem);
+	let currentTitle = currentItem["title"];
+	let currentDateTaken = currentItem["date_taken"];
+	// console.log(currentTitle + " " + currentDateTaken);
+}
+
+/*
+3. Create a new array called 'links'. Iterate through the items array in flickrObj and store the the links to each item in the 'links' array.
+*/
+
+
+for(let i = 0; i < flickrObjItemsLength; i++) {
+	let currentItem = flickrObjItems[i];
+	let currentLink = currentItem["link"];
+	let links = [currentLink];
```

Comment:
This is going to console.log all the links, but at the end of the loop, the links variable will contain only the final link. How could you use array.push() to store all the links in a single array?